packages/fury-adapter-apib-serializer/package.json

```diff
@@ -20,12 +20,12 @@
     "nunjucks": "^2.0.0"
   },
   "peerDependencies": {
-    "fury": "3.0.0-beta.9"
+    "fury": "3.0.0-beta.10"
   },
   "devDependencies": {
     "chai": "^4.1.2",
     "eslint": "^5.9.0",
-    "fury": "3.0.0-beta.9",
+    "fury": "3.0.0-beta.10",
```

Comment:
Strict dev-dep, but the apib-parser uses `^`. Is it intentional?

Reply:
I've updated fury-adapter-apib-parser and fury-adapter-swagger to use strict "3.0.0-beta.10" as we have everywhere else. Good catch, I missed that one. Fighting lerna here as it is trying to use `^` with major beta release which isn't safe as any future beta can introduce incompatibilities.

Although in our case it does not really matter because Fury is in same repository and that is used to fulfil the dependency which is developed along side all packages in this repo.